src/doc/trpl/dining-philosophers.md

```diff
@@ -243,15 +238,15 @@ fn eat(&self) {
 In Rust, methods take an explicit `self` parameter. That’s why `eat()` is a
 method, but `new` is an associated function: `new()` has no `self`. For our
 first version of `eat()`, we just print out the name of the philosopher, and
-mention they’re done eating. Running this program should give you the following
-output:
+mention that she’s done eating. Running this program should give you the
```

Comment:
This change from “they’re” to “she’s” is linguistically incorrect. (The presence of the word “that” probably fits more English locales than the lack of it, regardless of the word that follows.)